specification/2.0/README.md

```diff
@@ -1003,6 +1004,24 @@ The following examples shows a material that is defined using `pbrMetallicRoughn
 >| Occlusion | Model will appear brighter in areas that should be darker. |
 >| Emissive  | Model with lights will not be lit. For example, the headlights of a car model will be off instead of on. |
 
+### Alpha Coverage
+
+The `alphaMode` property defines how the alpha value of the main factor and texture should be interpreted. The alpha value is defined in the `baseColor` for metallic-roughness material model. 
+
+`alphaMode` can be one of the following values:
+* `OPAQUE` - The rendered output is fully opaque and any alpha value is ignored.
+* `MASK` - The rendered output is either fully opaque or fully transparent depending on the alpha value and the specified alpha cutoff value. This mode is used to simulate geometry such as tree leaves or wire fences.
+* `BLEND` - The rendered output is combined with the background using the normal painting operation (i.e. the Porter and Duff over operator). This mode is used to simulate geometry such as guaze cloth or animal fur. 
+
+ When `alphaMode` is set to `MASK` the `alphaCutoff` property specifies the cutoff threshold. If the alpha value is greater than or equal to the `alphaCutoff` value then it is rendered as fully opaque, otherwise, it is rendered as fully transparent. `alphaCutoff` value is ignored for other modes.
+
+The `doubleSided` property specified whether the material is double sided.
```

Comment:
This seems a bit out of place. Double sided has nothing to do with alpha coverage?